ui/src/components/pamFileList/components/fileManage/index.vue

```diff
@@ -291,9 +423,13 @@ const rowProps = (row: RowData) => {
         return message.error('暂不支持文件预览');
       }
 
-      mittBus.emit('change-path', { path: splicePath });
+      mittBus.emit('file-manage', { path: splicePath, type: ManageTypes.CHANGE });
       disabledBack.value = false;
     }
   };
 };
+
+onBeforeUnmount(() => {
+  unloadListeners();
+});
 </script>
```

Comment:
This is already quite well done code, but there are a few things that can be improved.

1. Use `computed`: This makes writing properties like `disabledBack`, `showDropdown` etc easier.
2. Import components earlier: You may want to import them in this script setup block. It should not be done outside.
3. Avoid magic string references (`@@-`) and use variable names as close to their actual purpose as possible (ex.`t("newFile")`).
4. Ensure consistency between template strings. For example, using `{...}` for multiple parts seems unnecessary since it's just repeating the same structure each time. Also add double quotes around literals instead of single ones.
```json5
import mittBus from "@/utils/mittBus.ts";
```
5. The `handleNewFolder` method needs input handling to make sure the proper arguments reach it correctly. In particular, `emit()` calls expect two parameters here while they only receive one argument directly when passed through the props object.
6. Check if you need to pass an object where every prop has its own unique name. If so, don't hard-code values into props, but instead create a custom computed property for those fields. 

The above list covers most important coding guidelines. Remember that TypeScript/JavaScript is often very strict about these rules in production applications, ensuring your component behaves according to intended usage.